frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -1095,6 +1095,10 @@ function ReaderBookmark:showBookmarkDetails(item)
         title = T(_("Bookmark details (%1/%2)"), item.idx, #item_table),
         text = text,
         text_type = "bookmark",
+        close_callback = function()
+            self:refreshBookmarkDetails()
+            UIManager:close(textviewer)
```

Comment:
Not needed, already closed in
https://github.com/koreader/koreader/blob/698fc35c3f1a10233cf69085a87fb912e2f79841/frontend/ui/widget/textviewer.lua#L384